src/App.js

```diff
@@ -1,8 +1,14 @@
 import "./App.css";
+import { Footer } from "./frontend/components/Footer/Footer";
+import { Navbar } from "./frontend/components/Navbar/Navbar";
+import { Homescreen } from "./frontend/screens/Homescreen/Homescreen";
```

Comment:
You can create an index.js file inside the components folder and import all the components there only. Make it one liner import here 
like - import { Footer, Navbar, Homescreen } from './frontend/components/index.js'

Reply:
yup gonna get that done in the next pr